dev/source/docs/mavlink-requesting-data.rst

```diff
@@ -159,7 +159,7 @@ Specifying Message Rates in a File
 
 At boot ArduPilot will populate the initial message intervals from files found in either ROMFS or in the filesystem.
 
-On ChibiOS-based boards the SD card will be searched for specially-named files in the APM subdirectory.
+On ChibiOS-based boards (with more than 1MB of flash) the SD card will be searched for specially-named files in the top directory.
```

Comment:
sorry I have never heard the root or root directory of the SD card called "top" directory
```suggestion
On ChibiOS-based boards (with more than 1MB of flash) the SD card will be searched for specially-named files in the  root directory, sometimes called top or base directory.
```

Reply:
Yes, "root" is better than "top" so I've changed it.  txs!